public/app/features/query/components/QueryEditorRow.tsx

```diff
@@ -35,10 +35,9 @@ interface Props {
   queries: DataQuery[];
   id: string;
   index: number;
-  timeRange?: TimeRange;
   dataSource: DataSourceInstanceSettings;
   onChangeDataSource?: (dsSettings: DataSourceInstanceSettings) => void;
-  onChangeTimeRange?: (timeRange: TimeRange) => void;
```

Comment:
Since we want to use a `RelativeTimeRange` if felt a bit weird to pass it to the `QueryEditorRow` since that type only makes sense in the alerting scenario. That is why I replaced this with a `renderExtras` function.